src/components/pages/account/InvestmentPlaceholderInfo.tsx

```diff
@@ -20,9 +20,10 @@ export type InvestmentPlaceholderInfoProps = {
 export default function InvestmentPlaceholderInfo({
   account,
 }: InvestmentPlaceholderInfoProps): JSX.Element {
-  const { data: investments, isPending } = useInvestments(
-    data => data.filter(d => account.childrenIds.includes(d.account.guid)),
+  const { data: investments = [], isPending } = useInvestments(
+    data => data.filter(d => d.account.path.startsWith(account.path)),
```

Comment:
path.startsWith is a good way to check the investment is a leaf inside this account tree.